pkg/tempopb/tempo.proto

```diff
@@ -110,6 +110,12 @@ message TraceSearchMetadata {
   uint32 durationMs = 5;
   SpanSet spanSet = 6; // deprecated. use SpanSets field below
   repeated SpanSet spanSets = 7;
+  map<string, ServiceStats> serviceStats = 8;
```

Comment:
Making this field non-nullable `[(gogoproto.nullable) = false]` to get `map[string]ServiceStats` unfortunately doesn't work, as then `jsonpb.Marshaler` always marshalls the `ServiceStats` struct to an empty JSON object `{}`, even when `ServiceStats` is not empty.